src/main/java/org/folio/circulation/domain/policy/lostitem/LostItemPolicy.java

```diff
@@ -53,20 +56,27 @@ public static LostItemPolicy from(JsonObject lostItemPolicy) {
       getActualCostFee(lostItemPolicy),
       getFeeRefundInterval(lostItemPolicy),
       getBooleanProperty(lostItemPolicy, "returnedLostItemProcessingFee"),
-      getChargeOverdueFineProperty(lostItemPolicy)
+      getChargeOverdueFineProperty(lostItemPolicy),
+      getAgedToLostAfterOverdueIntervalProperty(lostItemPolicy)
     );
   }
 
+  private static Period getAgedToLostAfterOverdueIntervalProperty(JsonObject policy) {
+    final JsonObject intervalJson = policy.getJsonObject("itemAgedLostOverdue");
+
+    return intervalJson != null ? Period.from(intervalJson) : null;
```

Comment:
What does `Period.from` return if the `JsonObject` is null?

If it is null, could this logic be consolidated?

Reply:
It returns a `Period` with a null interval and duration.

Reply:
Hmm, I wonder how that works. Maybe introducing a proper null object for periods might be something worth considering here, rather than having to have null checks in both the JSON interpretation and the domain.